rust/src/transformations/make_stable_expr/expr_cast/test.rs

```diff
@@ -0,0 +1,69 @@
+use std::env::set_var;
+
+use crate::domains::{AtomDomain, LazyFrameDomain};
+use crate::metrics::SymmetricDistance;
+use crate::transformations::make_stable_lazyframe;
+
+use super::*;
+
+#[test]
+fn test_make_expr_cast() -> Fallible<()> {
+    let lf_domain = LazyFrameDomain::new(vec![
+        SeriesDomain::new("String", AtomDomain::<String>::default()),
+        SeriesDomain::new("f64", AtomDomain::<f64>::default()),
+        SeriesDomain::new("i64", AtomDomain::<i64>::default()),
+        SeriesDomain::new("bool", AtomDomain::<bool>::default()),
```

Comment:
This test is great. I wonder if it would help readability to split it in four smaller tests, one fore each type? If there were a failure, smaller tests would be more informative, too: Is everything failing, or just one type.